concent_api/core/templates/admin/payment_report.html

```diff
@@ -0,0 +1,155 @@
+        <tr>
+            <th>Provider:</th>
+            <td colspan="2">
+                {% for address in  pair_report.provider_ethereum_addresses %}
+                    <a href="https://{{ eth_network }}etherscan.io/address/{{ address }}">{{ address }}</a>,
+                {% endfor %}
+                public key: {{ pair_report.provider.public_key }}
+            </td>
+        </tr>
+        <tr>
+            <th>Requestor:</th>
+            <td colspan="2">
+                {% for address in  pair_report.requestor_ethereum_addresses %}
+                    <a href="https://{{ eth_network }}etherscan.io/address/{{ address }}">{{ address }}</a>,
+                {% endfor %}
+                public key: {{ pair_report.requestor.public_key }}
+            </td>
+        </tr>
+
+        {#  Pending Responses table  #}
+        <tr>
+            <th rowspan="3">PendingResponses</th>
+            <th>SubtaskResultsSettled</th>
+            <td>
+            {% for subtask_results_settled in pair_report.pending_responses.subtask_results_settled %}
+                <p><a href="{% url 'admin:core_pendingresponse_change' subtask_results_settled.id %}">
+                - subtask_id: {{ subtask_results_settled.subtask.subtask_id }},
+                delivered: {{ subtask_results_settled.delivered }},
+                created_at: {{ subtask_results_settled.created_at }}</a>
+                </p>
+            {% endfor %}
+            </td>
+        </tr>
+        <tr>
+            <th>SubtaskResultsRejected</th>
+            <td>
+            {% for subtask_results_rejected in pair_report.pending_responses.subtask_results_rejected %}
+                <p><a href="{% url 'admin:core_pendingresponse_change' subtask_results_rejected.id %}">
+                    - subtask_id: {{ subtask_results_rejected.subtask.subtask_id }},
+                    delivered: {{ subtask_results_rejected.delivered }},
+                    created_at: {{ subtask_results_rejected.created_at }}</a>
+                </p>
+            {% endfor %}
+            </td>
+        </tr>
+        <tr>
+            <th>ForcePaymentCommitted</th>
+            <td>
+            {% for force_payment_committed in pair_report.pending_responses.force_payment_committed %}
+                <p><a href="{% url 'admin:core_pendingresponse_change' force_payment_committed.id %}">
+                    - delivered: {{ force_payment_committed.delivered }},
+                    created_at: {{ force_payment_committed.created_at }}</a>
+                </p>
+            {% endfor %}
+            </td>
+        </tr>
+
+        {#  Payments table  #}
+        <tr>
+            <th rowspan="5">Payments</th>
+            <th>Regular payments</th>
+            <td>
+            {% for payment in pair_report.payments.regular_payments %}
+                <p>
+                    <a href="https://{{ eth_network }}etherscan.io/address/{{ payment.provider }}">- provider: {{ payment.provider }}</a>,
```

Comment:
Same here, partial template and include.

Reply:
Done